changelog/2034.breaking.rst

```diff
@@ -0,0 +1,3 @@
+|particle_input| no longer enforces that |parameters| named
+``ionic_level`` are ions or neutral atoms. For equivalent behavior,
+name the parameter ``ion`` instead.
```

Comment:
This is technically a breaking change but it's pretty obscure behavior so no deprecation period is needed.

Reply:
We have a function called `ionic_levels`, but I don't think we use `ionic_level` as a parameter to a function anywhere. So, this is very unlikely to cause anyone any problems.